releasenotes/notes/add-upper-constraints-to-es-c2eb0b51c83e30e2.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    The elasticsearch role now respects upper constraints when installing
+    pip packages. The upper constriants file is defined in
+    ``rpcd/playbooks/group_vars/all.yml``.
```

Comment:
I think this is an old release note that hasn't been updated for a while. I'll remove the bits about the upper constraints file, there is no need for that information to be here.